README.md

```diff
@@ -25,7 +25,7 @@ search experience up and running in minutes.
 ### Features :+1:
 
 - **You know, for search** - Maintained by [Elastic](https://elastic.co), the team behind Elasticsearch.
-- **Speedy Implementation** - Build a complete search experience with a few lines of code.
+- **Speedy Implementation** - Build a complete, responsive search experience with a few lines of code.
```

Comment:
See above comment - if we want to keep it simple for now, I'm happy to lose the accessibility section for now and have this be `Build a complete, responsive, and accessible search experience with a few lines of code` :)